books/mindsBrainsAndScience.org

```diff
@@ -0,0 +1,29 @@
+#+TITLE: Minds, Brains and Science
+#+AUTHOR: John Searle
+#+DATE: 1984
```

Comment:
Our convention is put the date that you read it and/or the date that you are submitting it to Codex.